docs/structure.md

```diff
@@ -191,10 +191,27 @@
 
     === "JobTask"
 
-        {{ docstring_jobtask }}
-        <br>
-        Available functions, see also [**JobTask reference**](jobtask-reference.md):
-        {{ format_funcs(funcs_jobtask) }}
+        The JobTask class enables access to [results of a specific job task](../../analytics/).
```

Comment:
The link doesn't work.

Reply:
Got it! Working locally for me now. Marking resolved, but please let me know if there are still problems on the next review :)